src/navigator/IFrameNavigator.ts

```diff
@@ -684,10 +684,13 @@ export default class IFrameNavigator implements Navigator {
             let startUrl: string | null = null;
             if (startLink && startLink.href) {
                 startUrl = this.publication.getAbsoluteHref(startLink.href);
-
             }
 
             if (lastReadingPosition) {
+                const linkHref = this.publication.getAbsoluteHref(lastReadingPosition.href);
+                console.log(lastReadingPosition.href)
```

Comment:
Do you want an if (IS_DEV) on these two console.log statements?

Reply:
done.